src/primitives/transaction.h

```diff
@@ -413,6 +414,7 @@ class CTxIn
     CTxIn()
     {
         nSequence = SEQUENCE_FINAL;
+        m_is_pegin = false;
```

Comment:
this seems redundant, since you already initialized it at the declaration. (in fact, ``nSequence`` should probably also be initialized at declaration)